app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from __future__ import annotations
+
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum(elf_player_class.get_rating() for elf_player_class in team)
```

Comment:
why do u call variable elf_player_class, if this method could work with every class ?

Reply:
Probably i saw examples in readme and recognized it as a case scenario.So that was in my mind during coding process. Sorry about this kind of mistakes.They are stupid